test/functional/feature_assumeutxo.py

```diff
@@ -518,6 +519,28 @@ def check_dump_output(output):
         assert_equal(utxo_info['height'], SNAPSHOT_BASE_HEIGHT)
         assert_equal(utxo_info['bestblock'], snapshot_hash)
 
+        self.log.info("Check that getblockchaininfo returns information about the background validaiton process")
+        expected_keys = [
+            "snapshotheight",
+            "blocks",
+            "bestblockhash",
+            "mediantime",
+            "chainwork",
+            "verificationprogress"
+        ]
+        res = n1.getblockchaininfo()
+        assert "background" in res.keys()
```

Comment:
As this already is a rather too long test, it would be nice to move these assertions into a separate method, similarly as it done with check_tx_counts